.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Found a bug in Fluent UI React Native? Please let us know.
+---
+
+<!--
+Thanks for contacting us! We're here to help.
+Before you report an issue, check if it's been reported before:
+  * Search: https://github.com/microsoft/fluentui-react-native/search?type=Issues
+Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
+-->
+
+### Environment Information
+
+- **Package version(s)**: (fill this out, include which package manager you're using)
+- **OS version**: (fill this out if relevant)
+
+- **Platform**:
+  - [ ] iOS
+    - **Xcode version**:  (fill this out if relevant)
```

Comment:
can remove "fill this out if relevant" since the structure guarantees its relevance?